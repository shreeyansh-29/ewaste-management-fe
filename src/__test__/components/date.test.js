/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import toJson from "enzyme-to-json";
import Date from "../../components/date";

Enzyme.configure({adapter: new Adapter()});

it("test categoryCarousel", () => {
  expect.assertions(1);
  expect(toJson(shallow(<Date />))).toMatchSnapshot();
});
