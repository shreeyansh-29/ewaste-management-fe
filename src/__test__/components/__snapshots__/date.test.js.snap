// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test categoryCarousel 1`] = `<Component />`;
